### PR TITLE
More friendly rake code:analysis 

### PR DESCRIPTION
**Pull request by jeroig**
#### Description:

The project has a rake task that didn't appear in list task

---
#### Expected behavior:

`bundle exec rails -T`

Screen Shot 2021-04-06 at 17 28 08

### CODE REVIEW

**Review by martinjaimem — APPROVED** [review]

File: lib/tasks/code_analysis.rake [hunk 1]
```diff
@@ -1,6 +1,9 @@
-task code_analysis: :environment do
-  sh 'bundle exec brakeman . -z -q'
-  sh 'bundle exec rubocop .'
-  sh 'bundle exec reek app lib public spec tmp'
-  sh 'bundle exec rails_best_practices .'
+namespace :code do
+  desc 'Execute code quality'
```
Comment (santib) [inline lib/tasks/code_analysis.rake:2]:
```suggestion
  desc 'Run code quality tools'
```

**Review by santib — APPROVED** [review]

File: lib/tasks/code_analysis.rake (hunk 1 above)
Comment (horacio) [inline lib/tasks/code_analysis.rake:2]: What do you think of `Executes code quality checks`?

**Review by horacio — APPROVED** [review]
💪

File: lib/tasks/code_analysis.rake (hunk 1 above)
Comment (jeroig) [reply lib/tasks/code_analysis.rake]: Done! thanks @santib

**Review by juan-apa — APPROVED** [review]
👏

**Pull request merged**